include/aspect/material_model/melt_global.h

```diff
@@ -131,6 +131,7 @@ namespace aspect
         double compressibility;
         double melt_compressibility;
         bool include_melting_and_freezing;
+        double melting_rate_op_split;
```

Comment:
I am not happy about this name. If at all, it should be `melting_rate_for_operator_splitting`, but that is not good either. What about `melting_timescale`? That would be the inverse of the current parameter. Also the fact that in the description for the parameter file you constantly have to refer to the inverse of this parameter hints to the fact that we maybe should store the inverse instead. What do you think?

Reply:
I could also just call it `melting_rate`, I guess. 
I just though it would be more intuitive to give a rate, but maybe you're right and a time scale would make more sense. I guess it would be similar to a half life, so yes, I'll change it to `melting_time_scale`.